revisions/N0/src/text/dialog_3.asm

```diff
@@ -105,6 +105,7 @@ Dialog111::
     db "Kom og hent det,"
     db "hvis du tør!»   "
     db "Mester <skull>@"
+; Rart med «boksen» her? Det er jo direkte oversatt, bare skurrer litt. 
```

Comment:
ja, for det er i en kiste? kiste?

Reply:
ja, kiste i Mallegapet. Han har tatt hookshoten fra kista etter første boss. spå engelsk står det box da, så litt samme for meg. bare syns boks på norsk ikke passer helt, kan finne et annet ord

Reply:
endrer du til kista? i så fall er alt okei tenker jeg. jeg klarte ikke å legge inn suggestion på den linja.

Reply:
ja d r fiksa